README.md

```diff
@@ -1,6 +1,66 @@
 Support for detecting and being notified when an object is mutated.
 
+An observable, simply put, is a continuous stream of events over time.
+
+### What's for
+
+You can use this library to process asynchronous streams of data for logging, data binding or custom changes to them. You can think of it as the former TC39 proposal Object.observe() feature that got deprecated.
+
+### How
+
 There are two general ways to detect changes:
 
 * Listen to `Observable.changes` and be notified when an object changes
 * Use `Differ.diff` to determine changes between two objects
+
+### Examples
+
+Operations on Lists:
```

Comment:
Added one for demonstrating an observable list. Afraid that it might be too verbose though, let me know what you think.